server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -271,13 +267,11 @@ export function configureWebSocketServices(
             const connectVersions = message.versions ? message.versions : ["^0.1.0"];
             const version = selectProtocolVersion(connectVersions);
             if (!version) {
-                // eslint-disable-next-line prefer-promise-reject-errors
-                return Promise.reject({
-                    code: 400,
-                    message: `Unsupported client protocol. ` +
-                        `Server: ${protocolVersions}. ` +
-                        `Client: ${JSON.stringify(connectVersions)}`,
-                });
+                throw new NetworkError(
+                    400,
+                    // eslint-disable-next-line max-len
+                    `Unsupported client protocol. Server: ${protocolVersions}. Client: ${JSON.stringify(connectVersions)}`,
```

Comment:
No privacy concerns with this data being logged?

Reply:
This information _should_ just be a bunch of version numbers unless the client changes the driver to send PII as "version" which should never happen